test/unit/paged-sync.test.ts

```diff
@@ -48,6 +53,7 @@ describe('paged-sync', () => {
         {
           initial: true,
           content_type: 'id',
+          // @ts-ignore
```

Comment:
why was this `@ts-ignore` added?

Reply:
It's a test which checks if request is rejected for invalid content type and type. New types force you to only use `Entry` as type if `content_type` is provided, therefore I need to force TS to ignore the invalid value to run the test.

Reply:
Oh I understand it more now, thank you!